doc/source/serve/scaling-and-resource-allocation.md

```diff
@@ -0,0 +1,160 @@
+The underlying cloud provider will then respond by adding more nodes.
+Similarly, when Ray Serve scales down and terminates some replica actors, it will try to do so in a way that results in the most nodes having no Ray actors or tasks running on them, at which point the Ray autoscaler will remove those nodes.
+:::
+
+To learn about the architecture underlying Ray Serve Autoscaling, see {ref}`serve-autoscaling-architecture`.
+
+### `autoscaling_config` parameters
+
+There are several user-specified parameters the autoscaling algorithm takes into consideration when deciding the target number of replicas for your deployment:
+
+**min_replicas[default_value=1]**: The minimum number of replicas for the deployment. ``min_replicas`` will also be the initial number of replicas when the deployment is deployed.
+:::{note}
+Ray Serve Autoscaling allows the `min_replicas` to be 0 when starting your deployment; the scale up will be started when you start sending traffic. There will be a cold start time as the Ray ServeHandle waits (blocks) for available replicas to assign the request.
+:::
+**max_replicas[default_value=1]**: Max replicas is the maximum number of replicas for the deployment. Ray Serve Autoscaling will rely on the Ray Autoscaler to scale up more nodes when the currently available cluster resources (CPUs, GPUs, etc) are not enough to bring up more replicas.
+
+**target_num_ongoing_requests_per_replica[default_value=1]**: The config is to maintain how many ongoing requests are expected to run concurrently per replica at most. If the number is lower, the scale up will be done more aggressively.
+:::{note}
+- It is always recommended to load test your workloads. For example, if the use case is latency sensitive, you can lower the `target_num_ongoing_requests_per_replica` number to maintain high performance.
+- Internally, the autoscaler will decide to scale up or down by comparing `target_num_ongoing_requests_per_replica` to the number of `RUNNING` and `PENDING` tasks on each replica.
+- `target_num_ongoing_requests_per_replica` is only a target value used for autoscaling (not a hard limit), the real ongoing requests number can be higher than the config.
+:::
+
+**downscale_delay_s[default_value=600.0]**: The config is to control how long the cluster needs to wait before scaling down replicas.
+
+**upscale_delay_s[default_value=30.0]**: The config is to control how long the cluster need to wait before scaling up replicas.
+
+:::{note}
+`downscale_delay_s` and `upscale_delay_s` are to control the frequency of doing autoscaling work. For example, if your use case takes a long time to do initialization work, you can increase `downscale_delay_s` to make the downscaling happen slowly.
+:::
+
+**smoothing_factor[default_value=1]**: The multiplicative factor to speedup/slowdown each autoscaling step. For example, when the use case has high traffic volume in short period of time, you can increase `smoothing_factor` to scale up the resource quickly.
```

Comment:
```suggestion
**smoothing_factor[default_value=1]**: The multiplicative factor to speed up or slow down each autoscaling step. For example, when the application has high traffic volume in a short period of time, you can increase `smoothing_factor` to scale up the resource quickly.
```

Does this mean that a high `smoothing_factor` results in choppier autoscaling? It sounds like the higher it is, the quicker the autoscaler scales the deployment.

Reply:
Yes, this is backwards.  I'll confirm and fix it

Reply:
Actually the description is correct, the name is bad.  Maybe it should have been called `gain`.  I added a clarifying comment: "You can think of this as a "gain" factor to amplify the response of the autoscaling algorithm."